app/controllers/catalog_controller.rb

```diff
@@ -489,4 +489,23 @@ def index
   rescue ActionController::BadRequest
     render file: Rails.public_path.join('x400.html'), layout: true, status: :bad_request
   end
+
+  # @see Blacklight::Catalog#facet
```

Comment:
can you add a github link to this comment? make sure to hit `y` to get the canonical link location, since it's a bit different in master. it might be worth noting that this changes with blacklight version 7.